### PR TITLE
config for oauth, mvc test

### PR DESCRIPTION
**Pull request by lor6**

### CODE REVIEW

File: spring-security-oauth-server/src/test/java/org/baeldung/test/OAuthMvcTest.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+import org.springframework.web.context.WebApplicationContext;
+
+import org.springframework.boot.json.JacksonJsonParser;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.baeldung.mvc.config.AuthorizationServerMvcApplication;
+
+@RunWith(SpringRunner.class)
+@WebAppConfiguration
+@SpringBootTest(classes = AuthorizationServerMvcApplication.class)
+public class OAuthMvcTest {
```
Comment (KevinGilmore) [inline spring-security-oauth-server/src/test/java/org/baeldung/test/OAuthMvcTest.java]: I get SQL syntax errors while trying to run these tests -- H2 can't create the tables:
...
create table if not exists oauth_client_token ( token_id VARCHAR(255), token LONG VARBINARY, authentication_id VARCHAR(255) PRIMARY KEY, user_name VARCHAR(255), client_id VARCHAR(255) )

nested exception is org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "CREATE TABLE IF NOT EXISTS OAUTH_CLIENT_TOKEN ( TOKEN_ID VARCHAR(255), TOKEN LONG VARBINARY[*], AUTHENTICATION_ID VARCHAR(255) PRIMARY KEY, USER_NAME VARCHAR(255), CLIENT_ID VARCHAR(255) ) "; expected "RAW, (, FOR, UNSIGNED, NOT, NULL, AS, DEFAULT, GENERATED, NOT, NULL, AUTO_INCREMENT, BIGSERIAL, SERIAL, IDENTITY, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, PRIMARY, UNIQUE, NOT, NULL, CHECK, REFERENCES, ,, )"; 
SQL statement:
create table if not exists oauth_client_token ( token_id VARCHAR(255), token LONG VARBINARY, authentication_id VARCHAR(255) PRIMARY KEY, user_name VARCHAR(255), client_id VARCHAR(255) ) [42001-193]

File: spring-security-oauth-server/src/main/java/org/baeldung/mvc/config/EmployeeController.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+@Controller
+public class EmployeeController {
+
+    private List<Employee> employees = new ArrayList<>();
+
+    @RequestMapping(value = "/employee")
+    @ResponseBody
+    public Employee getEmployee(@RequestParam String email) {
+        Employee result = employees.stream().filter(x -> email.equals(x.getEmail())).findAny().orElse(null);
```
Comment (pivovarit) [inline spring-security-oauth-server/src/main/java/org/baeldung/mvc/config/EmployeeController.java]: "email.equals(x.getEmail())" we might want to reverse it, email can be null, right?

File: spring-security-oauth-server/src/main/java/org/baeldung/mvc/config/EmployeeController.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        return result;
```
Comment (pivovarit) [inline spring-security-oauth-server/src/main/java/org/baeldung/mvc/config/EmployeeController.java]: Let's inline it, storing result temporarily does not give us anything

File: spring-security-oauth-server/src/test/java/org/baeldung/test/OAuthMvcTest.java (hunk 1 above)
Comment (pivovarit) [inline spring-security-oauth-server/src/test/java/org/baeldung/test/OAuthMvcTest.java]: @KevinGilmore is this solved already?

**Pull request merged**